public/app/plugins/panel/state-timeline/StateTimelinePanel.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useCallback, useMemo, useRef, useState } from 'react';
 
 import { CartesianCoords2D, DashboardCursorSync, DataFrame, FieldType, PanelProps } from '@grafana/data';
+import { getLastStreamingDataFramePacket } from '@grafana/data/src/dataframe/StreamingDataFrame';
```

Comment:
nit: import from `@grafana/data` instead?

Reply:
same, not exported